package.json

```diff
@@ -57,6 +57,18 @@
           ],
           "language": "ts"
         }
+      },
+      "Or with --esModuleInterop enabled you can do the following:",
+      {
+        "code": {
+          "content": [
+            "import Anser from 'anser';",
+            "const txt = \"\\u001b[38;5;196mHello\\u001b[39m \\u001b[48;5;226mWorld\\u001b[49m\";",
+            "console.log(Anser.ansiToHtml(txt));",
+            "// <span style=\"color:rgb(255, 0, 0)\">Hello</span> <span style=\"background-color:rgb(255, 255, 0)\">World</span>"
+          ],
+          "language": "ts"
+        }
```

Comment:
Wasn't sure whether to update the README as well since there's a comment at the top of the README that says:
```
<!-- Please do not edit this file. Edit the `blah` field in the `package.json` instead. If in doubt, open an issue. -->
```